Fabric.Authorization.API/DependencyInjection/InMemoryConfigurator.cs

```diff
@@ -14,5 +15,11 @@ protected override TinyIoCContainer.RegisterOptions RegisterDatabaseContext(Tiny
         {
             return container.Register<IAuthorizationDbContext, InMemoryAuthorizationDbContext>();
         }
+
+        protected override TinyIoCContainer.RegisterOptions RegisterEDWDatabaseContext(TinyIoCContainer container)
+        {
+            throw new NotImplementedException("There is no In memory EDW Database context.");
```

Comment:
If we don't provide an InMemoryEdwContext, then our in memory integration tests will not work properly.

Reply:
We thought that.  We will tackle this after we get the app working as a whole.

Reply:
I have wired up one.